### ISSUE TITLE
Data type String lists, Sorting Edges, Node Label display

### ISSUE THREAD

**Issue by reversingentropy** [gephi/gephi#2558]
Hi all, I am trying to visualize a network between individuals based on common keywords.

1) Each individual has a list of string keywords. Using a list of strings as a node attribute, is there a way to partition the network and filter nodes based on strings in the string list? It seems that string lists are not recognized as string lists but strings.

Search 1st Degree as a column of string lists.
<img width="303" alt="image" src="https://user-images.githubusercontent.com/68902907/179145975-bc23d1bc-5aeb-4309-936a-9ec9a3bcc11a.png">

String lists treated as string
<img width="163" alt="image" src="https://user-images.githubusercontent.com/68902907/179146023-ec038bc9-5b5c-46f1-8fd9-3a29c4e226ec.png">

2) If I partition by edge labels, it is sorted by value counts in descending order. Is it possible to sort the values alphabetically?

<img width="106" alt="image" src="https://user-images.githubusercontent.com/68902907/179146632-74bb22aa-4dd1-4844-988f-89927ff53e34.png">

3) Is it possible to view more selected node labels only after pointing at a selected node?

**Comment by mbastian** [comment]
Hi @reversingentropy 

1. The #2061 issue already captures this gap. Currently, list types are not decomposed by the partition module so that you can't manipulate or filter based on the list elements.

The way values are shown in the filter panel is however problematic, and I created an separate issue for that #2569.

2. No, sorting is hardcoded to be in descending order by frequency. I would appreciate if you could create a feature request issue just for that.

3. Sorry I do not understand your question.
